udiskie/mount.py

```diff
@@ -84,6 +84,7 @@ def __init__(self, udisks,
         self._mount_options = mount_options or (lambda device: None)
         self._ignore_device = ignore_device or FilterMatcher([], False)
         self._ignore_device._filters += [
+            IgnoreDevice({'should_automount': False, 'ignore': True}),
```

Comment:
it'd make more sense to move the newly added ignore rule down by one line:

logically it doesn't make much sense to check for the ignore hint, if the device is not even a block device. But it should work either way.